@navikt/core/react/src/link/Link.tsx

```diff
@@ -4,6 +4,18 @@ import { OverridableComponent } from "../util/OverridableComponent";
 
 export interface LinkProps
   extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
+  /**
+   * Inverts when the underline appears. If this is true,
+   * the underline does not appear by default, but does appear when the link is hovered.
+   * This makes it more suitable for use when inlined in text.
+   * @default false
+   */
+  "remove-underline"?: boolean;
```

Comment:
Hvis vi etterhvert gjør noen oppdateringer på typo-komponentene våre vil de få props som `weight`, `variant` og kanskje `underline`. For å da matche disse ville kanskje `underline` som boolean fungert?
```jsx
/* @default true */
underline?: boolean;
```